src/frontend/apps/e2e/__tests__/app-desk/teams-delete.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from '@playwright/test';
+
+import { addNewMember, createTeam, keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Teams Delete', () => {
+  test('it deletes the team when we are owner', async ({
+    page,
+    browserName,
+  }) => {
+    await createTeam(page, 'team-update-name', browserName, 1);
+
+    await page.getByLabel(`Open the team options`).click();
+    await page.getByRole('button', { name: `Delete the team` }).click();
+    await page.getByRole('button', { name: `Confirm deletion` }).click();
+    await expect(page.getByText(`The team has been removed.`)).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: `Create a new team` }),
+    ).toBeVisible();
+  });
+
+  test('it deletes the team when we are admin', async ({
```

Comment:
nitpicking: `'it cannot delete the team when we are admin'`

Reply:
[fixup! 🛂(app-desk) add TeamActions component](https://github.com/numerique-gouv/people/pull/148/commits/da4456fb3ae86356220a8c96357d6c650d004a77)